packages/ra-no-code/src/ResourceBuilderContext/useResourceBuilder.ts

```diff
@@ -0,0 +1,4 @@
+import { useContext } from 'react';
+import { ResourceBuilderContext } from './ResourceBuilderContext';
+
+export const useResourceBuilder = () => useContext(ResourceBuilderContext);
```

Comment:
not fan of the name "builder", since it's also used at runtime

Reply:
It still builds a component right ? `useResourceInferrer`? You have something better?

Reply:
This one does not exist anymore